src/main/java/com/metabug/persistence/model/Ticket.java

```diff
@@ -0,0 +1,35 @@
+package com.metabug.persistence.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "ticket")
+public class Ticket {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    private long id;
+
+    @Column(name = "title")
+    private String title;
+
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "author_id")
+    private UUID authorId;
```

Comment:
shouldn't it be private User author with ManyToOne annotation?

Reply:
One ticket can have many authors?

Reply:
No, but one author can have many tickets - that's why i suggested ManyToOne annotation (many tickets - one author).

Reply:
This is task for next commit.